users/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from django.conf import settings
+
+class Users(models.Model):
+    LOGIN_USERID = "user"
+    LOGIN_KAKAO = "kakao"
+    LOGIN_GOOGLE = "google"
+
+    LOGIN_CHOICES = (
+        (LOGIN_USERID, "user"),
+        (LOGIN_KAKAO, "Kakao"),
+        (LOGIN_KAKAO, "google"),
+    )
+
+    name = models.CharField(max_length=255,null = True)
+    user = models.CharField(max_length=255)
+    password = models.CharField(max_length=255)
+    email = models.EmailField(max_length=255,null=True,blank=True)
+    phonenumber = models.IntegerField(null=True,blank=True)
+    address = models.CharField(max_length=255,null=True,blank=True) 
+    genderid = models.ForeignKey('Genders',on_delete = models.CASCADE, null = True)
```

Comment:
gender 라고만 선언해도 _id를 붙여주기 때문에 하기와 같이 수정 부탁 드립니다.
```suggestion
    gender = models.ForeignKey('Genders',on_delete = models.CASCADE, null = True)
```